src/java.desktop/share/native/libharfbuzz/hb.hh

```diff
@@ -103,20 +103,20 @@
 #pragma GCC diagnostic warning "-Wdisabled-optimization"
 #pragma GCC diagnostic warning "-Wdouble-promotion"
 #pragma GCC diagnostic warning "-Wformat=2"
+#pragma GCC diagnostic warning "-Wformat-signedness"
 #pragma GCC diagnostic warning "-Wignored-pragma-optimize"
```

Comment:
Reading these changes makes me think we should re-check if all the DISABLED_WARNINGS we have for
harbuzz in Awt2DLibraries.gmk (see lines 466-470 in that file) are still needed.
In particular for gcc + clang (ie windows and mac builds).
Can you comment them out (in that file) and test the build ? Any that are now un-needed can be removed.

Reply:
@prrace Sure Phil. I'll comment it out and check if there are any un-needed flags in DISABLED_WARNINGS section.

Reply:
@prrace Checked by commenting out the HARFBUZZ_DISABLED_WARNINGS section in Awt2DLibraries.gmk, all the currently listed warnings (for specific platform compilers) are required else the build fails.